### PR TITLE
Updates to Agile Adaptation section

### PR DESCRIPTION
**Pull request by tsemana**
Adding Product owner to project roles, 2-week iteration details, Trello vs Jira in tracking section, and BDD/TDD validation in workflow.

Updates to meetings and splitting discovery coming up.

### CODE REVIEW

File: docs/ProjectManagement-AgileAdaptation.md [hunk 1]
```diff
@@ -9,42 +9,59 @@ Our project management process is a form of Agile that has been adapted (and con
 
 Each project has the following roles:
 
-* Project lead: responsible for all things. We take turns being project lead. The project lead runs the [project meetings](#meetings)
-* Developer or designer: anyone else who’s working on the project.
+* Product owner: a member of the client team. Prioritizes product scope and features. Works with development team to fully define scope of upcoming work, reviews and signs off on completed features in a timely manner.
+* Project lead: a member of the technical team. Responsible for all things. We take turns being project lead. The project lead runs the [project meetings](#meetings)
+* Developer or designer: anyone else on the technical team who’s working on the project.
+* Delivery manager: keeps an eye on schedule, scope and workflow. Liase with Product owner on budget, schedule and scope decisions.
 * Project advisor: someone who’s not actively working on the project but keeps an eye on things and helps out where needed.
```
Comment (tsemana) [inline docs/ProjectManagement-AgileAdaptation.md]: @pauldowman we don't formally assign this role anymore.  Should we re-initiate it or remove?

**Comment by arrygoo** [comment]
I have no comment on the Jira section, but the rest looks really good to me!

**Review by arrygoo — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by rdickert — APPROVED** [review]
This looks great. I don't have any changes to suggest

**Review by pauldowman — APPROVED** [review]

**Comment by pauldowman** [comment]
Deployed.

**Pull request merged**